src/App.vue

```diff
@@ -7,6 +7,7 @@
             <sui-dropdown-header>{{$t('menu.games')}}</sui-dropdown-header>
             <router-link class="item" to="/questions">{{$t('menu.questions')}}</router-link>
             <router-link class="item" to="/logos">{{$t('menu.logos')}}</router-link>
+            <router-link class="item" to="/nutritions">{{$t('menu.nutritions')}}</router-link>
```

Comment:
https://github.com/openfoodfacts/openfoodfacts-hungergames/blob/master/src/i18n/common.json
Translation to be added in there